images.go

```diff
@@ -50,6 +50,22 @@ var imagesCommand = &cli.Command{
 }
 
 func imagesAction(clicontext *cli.Context) error {
+	filters := ""
+	if clicontext.NArg() > 0 {
+		slashCount := strings.Count(clicontext.Args().First(), "/")
+		switch slashCount {
+		case 0:
+			filters = fmt.Sprintf("name==docker.io/library/%s", clicontext.Args().First())
+			break
+		case 1:
+			filters = fmt.Sprintf("name==docker.io/%s", clicontext.Args().First())
+			break
+		default:
+			filters = fmt.Sprintf("name==%s", clicontext.Args().First())
+			break
+		}
```

Comment:
This doesn't work with images out of Docker Hub.

Please see rmi.go.

Reply:
Also, please use real name for signing.

Reply:
done. pls. check!

Reply:
You should return `err` for a badly formatted image name ( err != nil).

Reply:
- should error out when NArg > 2
- please squash commits